deploy/olm-catalog/ibm-block-csi-operator/ibm-block-csi-operator.v1.1.0.clusterserviceversion.yaml

```diff
@@ -7,7 +7,7 @@ metadata:
     capabilities: "Basic Install"
     categories: "Storage,Cloud Provider"
     certified: "true"
-    containerImage: ibmcom/ibm-block-csi-operator:1.1.0
+    containerImage: registry.connect.redhat.com/ibm/ibm-block-csi-operator:1.1.0
```

Comment:
what is the difference?

Reply:
`ibmcom` is DockerHub (for community) and the second is for Red Hat.
but I shouldn't change it here:
https://github.com/IBM/ibm-block-csi-operator/pull/66
will revert this